book/src/ui_usage.md

```diff
@@ -87,6 +87,82 @@ Siren provides the ability to exit/withdraw your validators via the validator ma
 
 ![exit](imgs/ui-val-exit.png)
 
+### Deposit and Import new Validators
+
+Siren's deposit flow aims to create a smooth and easy process for depositing and importing a new Lighthouse validator. The process is seperated into 6 main steps:
+
+#### Validator Setup
+
+- First, select the number of validators you wish to create, ensuring you connect a wallet with sufficient funds to cover each validator deposit. For each validator candidate, you can set a custom name and optionally enable the `0x02` withdrawal credential flag, which indicates to the deposit contract that the validator will compound and have an increased `MAX_EFFECTIVE_BALANCE`.
+
+![deposit-step-1](imgs/ui-dep-1.png)
+
+#### Phrase Verification
+
+- Enter a valid mnemonic phrase to generate corresponding deposit JSON and keystore objects. This is a sensitive step; copying and pasting your mnemonic phrase is not recommended. This information is never stored or transmitted through any communication channel.
+
+![deposit-step-2](imgs/ui-dep-2.png)
+
+#### Mnemonic Indexing
+
+- The mnemonic index is as important as the mnemonic phrase; reusing existing or exited indices may result in invalid deposits. Each index combined with the mnemonic phrase generates a deterministic public key, which Siren validates by checking against the Beacon Node. Since newly submitted deposits may not immediately appear on the Beacon Node, Siren provides BeaconCha links for secondary confirmation.
```

Comment:
Should we use the actual url here instead?

```suggestion
- The mnemonic index is as important as the mnemonic phrase; reusing existing or exited indices may result in invalid deposits. Each index combined with the mnemonic phrase generates a deterministic public key, which Siren validates by checking against the Beacon Node. Since newly submitted deposits may not immediately appear on the Beacon Node, Siren provides beaconcha.in links for secondary confirmation.
```